### DISCUSSION TITLE
Broken link on website

### DISCUSSION THREAD

**Discussion by gamebeaker** [opensubsonic/open-subsonic-api#114] [General]
On this site: https://opensubsonic.netlify.app/docs/opensubsonic-changes/#non-breaking-changes
Is the link for non breaking changes broken: https://opensubsonic.netlify.app/opensubsonic/changes/

**Comment by Tolriq** [comment]
Yes there's an additional "s" at the end, please make a PR.

**Comment by gamebeaker** [comment]
Closing as it looks like the pull request #115 gets accepted.
